.github/workflows/master_release.yml

```
@@ -22,8 +22,8 @@ env:
   GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
   BASE_BUILD_NUMBER: 10000
   # Release version has to start with real version (7.4.0-....) for the docker image to build successfully.
-  RELEASE_VERSION: 23.1.0-A48
-  DEVELOPMENT_VERSION: 23.1.0-A49-SNAPSHOT
+  RELEASE_VERSION: 23.1.0-A49
+  DEVELOPMENT_VERSION: 23.1.0-SNAPSHOT
```

```suggestion
  DEVELOPMENT_VERSION: 23.1.0-A50-SNAPSHOT
```

Hopefully won't be needed :)